legal-api/src/legal_api/services/namex.py

```diff
@@ -69,6 +69,11 @@ def query_nr_number(identifier: str):
             'Authorization': 'Bearer ' + token
         })
 
+        # Return if not successful so downstream can handle the error
+        if nr_response.status_code != 200:
+            return nr_response
+
+        # Return the json if successful
         return nr_response.json()
```

Comment:
I think converting this to json is the reason the name request GET endpoint is erroring out.  In looking at the legal-api logs, it looks like an error is being thrown from `namerequest.py` on the `if nr_response.status_code == 404:` line.  My guess is that since the `nr_response` object is being converted to json here, the get function in `namerequest.py` triggers the error we are seeing for retrieving NRs in DEV right now.

```2021-03-29 21:10:14,050 - legal_api - ERROR in namerequest:namerequest.py:46 - get: 'dict' object has no attribute 'status_code'
2021-03-29 21:10:14,058 - legal_api - ERROR in app:app.py:1891 - log_exception: Exception on /api/v1/nameRequests/NR 1637549 [GET]
```

Reply:
You bet! 
This pr is to address that error, as it was converting it later as well in the query_nr_number method.
So i updated that methods error handling and removed the second conversion, since we needed it to be json in a few places it made sense to apply it at the source.

Reply:
Line 44 in `namerequest.py` is the fix for this.

Reply:
Definitely open to any suggestions on solutions tho!